mappings/net/minecraft/block/MapColor.mapping

```diff
@@ -64,6 +64,7 @@ CLASS net/minecraft/class_3620 net/minecraft/block/MapColor
 	FIELD field_25706 DARK_AQUA Lnet/minecraft/class_3620;
 	FIELD field_25707 DARK_DULL_PINK Lnet/minecraft/class_3620;
 	FIELD field_25708 BRIGHT_TEAL Lnet/minecraft/class_3620;
+	FIELD field_33533 IRON Lnet/minecraft/class_3620;
```

Comment:
Hum, maybe a more precise name would be better, to make a difference with IRON_GRAY (used by regular iron blocks)

Reply:
I chose this name in order to be consistent with `MapColor.GOLD`.

Reply:
Gold is a color, iron is not.

Reply:
I think iron is fine as a color name.

Reply:
perhaps `RAW_IRON_ORANGE` or `PALE_BEIGE`.
Iron is not descriptive at all, it cannot be a color name.

Reply:
For reference, this is the color:
![image](https://user-images.githubusercontent.com/4873274/114765506-3996a900-9d33-11eb-85ab-67d408d6ff3c.png)

Reply:
It is definitely not orange and too pinkish to be beige. In my opinion, this color is the distinctive iron color.

Reply:
Iron in an MC context is gray.
This is more the color of the speckles on iron ore, but it is not the color of iron.

Reply:
At the very least `RAW_IRON_PINK` so it follows the naming conventions of all the other colors.

Reply:
```suggestion
	FIELD field_33533 IRON_PINK Lnet/minecraft/class_3620;
```
Since we already have `IRON_GRAY`

Reply:
I think `RAW_IRON_PINK` is good.

```suggestion
	FIELD field_33533 RAW_IRON_PINK Lnet/minecraft/class_3620;
```